tests/functional/features/sync_wrap.feature

```diff
@@ -22,3 +22,9 @@ Feature: Sync-wrap failure modes
         When the rate limit is tripped with sync-wrap polling
         Then I get a 503 HTTP response
         And returns a helpful error message
+
+    Scenario: The rate limit is tripped when POSTing new Patients (>1tps)
```

Comment:
The rate limit in SpikeArrest.PatientCreate is 5 TPS (300 TPM). This states 1 TPS?

Reply:
good spot, this needs changing